composer.json

```diff
@@ -92,7 +92,10 @@
     "config": {
         "preferred-install": "dist",
         "sort-packages": true,
-        "process-timeout": 3000
+        "process-timeout": 3000,
+        "allow-plugins": {
+            "phpstan/extension-installer": false
```

Comment:
Just curious why this was needed. BTW we'll fix the stuff Stan is complaining about in another commit. Usually after a while he starts to find more stuff for us to fix.

Reply:
my bad, I didn't intend to check in this change. I'll remove it.